app/react/Library/reducers/exportReducer.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from 'redux';
+import createReducer from 'app/BasicReducer';
+
+export default combineReducers({
+  exportProcessing: createReducer('exportProcessing', false),
```

Comment:
I'm wondering if this should be better namespaced?  Perhaps exportCSVProcessing?  or exportLibraryProcessing?  Also, there should be consistent naming across the store, because the exportEntity store entry point is missleading.  This is exportSearchResults, or exportLibrary.  ExportEntity should be a 'single' entity export and reserved for a different flow.

Reply:
Yes, makes sense. I'll take care of it.